R/html_document.R

```diff
@@ -78,6 +99,54 @@
 #'@param extra_dependencies,... Additional function arguments to pass to the
 #'  base R Markdown HTML output formatter \code{\link{html_document_base}}
 #'@return R Markdown output format to pass to \code{\link{render}}
+#'@section Highlighting:
+#'
+#'  Different highlighting engines can be used with a R Markdown HTML documents:
```

Comment:
I suggest making this a bulleted list. Maybe something like:

There are three highlighting engines available to HTML documents:

* highlightjs, which does highlighting in the browser, using javascript.
* pandoc, which uses pandoc
* downlite, which uses an R package (and only applies to R code)

Reply:
I used `\describe` to create the list instead of just bullet. That is clearer to separate the 3 engines.